README.md

```diff
@@ -1,45 +1,102 @@
-- Active
-- Work in Progress (WIP)
-- Draft
-- Last Call
-- Accepted 
-- Final
+Alternative statuses include: 
 
-Exceptional statuses:
-- Deferred: A core HIPs that have been put off for a future hard fork. 
-- Rejected: An HIP that is fundamentally broken or a core HIP that was rejected by the Core Devs.
-- Superseded: An HIP which was previously final but is no longer considered state-of-the-art.
+   - Deferred: HIPs with this status has been put off to be discussed again at a later date. 
+
+   - Rejected: HIPs with this status are either: fundamentally broken, not accepted by the community, or not accepted by the Hedera Governing Council. 
+
+   - Superseded: HIPs with this status were previously at some stage of acceptance, but is no longer considered state-of-the-art. 
+
+The status change of each HIP is requested by the HIP author and it is to be reviewed by the HIP editors. To update the status, use a pull request. 
+
+For convenience, always include a link for people to continue discussing your HIP or see the conversations that have already taken place. 
+
+## Before filling your HIP
+
+#### Step 1)
+Evaluate your idea: consider why you’d like to request changes or improvements, and how it benefits the Hedera Hashgraph community.  
+
+#### Step 2) 
+Thoroughly look through those proposals already submitted to ensure there are no duplicates. 
+
+#### Step 3) 
+Ask the Hedera Hashgraph community first if your idea is original, or has already been through the HIP process. 
+
+#### Step 4) 
+Reevaluate your proposal to ensure sure the idea is applicable to the entire community and not just to one particular author, application, project, or protocol. 
+
+If you're still here and think you can suggest an improvement, please go ahead and file an issue; we love contributions!
+
+##### Note 
+An excellent place to discuss your proposal and get feedback is in the [issues section of this repository](https://github.com/hashgraph/HIPs/issues), or on [hashgraph.org](https://hashgraph.org), a community forum dedicated to discussing hashgraph; there you can start formalizing the language around your HIP and ensuring it has broad community support. 
+
+## Disclaimer(s): 
+
+These improvement proposals are intended to be the community voice for changes, but the implementation of Hedera Hashgraph is left to the engineers working at Hedera, and decisions regarding the platform and codebase strategy or future roadmap are ultimately made by the Hedera Governing Council. At some point, Hedera may have their own improvement made separately from this community managed repository, and may make changes not outlined in an HIP. 
+
+These proposals and discussions have no effect regarding private (permissioned) implementations of the Hashgraph consensus algorithm; additionally, this repository and it’s contents are ran by the Hedera Hashgraph community, which means they do not necessarily reflect the views and opinions of Hedera Hashgraph LLC. 
+
+## Contributors 
```

Comment:
What is the purpose of the contributors section? People who've contributed to the README? People who've contributed PRs? I think it can probably be left off.

Reply:
No intention really, I believe it was intended to be contributors to HIPs - the Hashing Systems logo was already on the bottom and wanted to provide more context. Happy to remove